ckanext/iati_generator/blueprint/public_iati.py

```diff
@@ -0,0 +1,42 @@
+from flask import Blueprint, abort
+
+from ckan.plugins import toolkit
+from ckan import model
+from ckanext.iati_generator.models.iati_files import IATIFile
+from ckanext.iati_generator.models.enums import IATIFileTypes
+
+public_iati_blueprint = Blueprint("iati_public", __name__, url_prefix="/iati")
+
+
+@public_iati_blueprint.route("/<namespace>/<filename>")
+def serve_public_iati(namespace, filename):
+    """
+    Public endpoint that serves or redirects IATI XML files.
+    Example:
+        /iati/bcie/organization.xml
+        /iati/bcie/activities.xml
+    """
+    # Map filename -> enum type
+    if filename.lower() == "organization.xml":
+        file_type = IATIFileTypes.ORGANIZATION_MAIN_FILE.value
+    elif filename.lower() == "activities.xml":
```

Comment:
```suggestion
    if filename == "organization.xml":
        file_type = IATIFileTypes.ORGANIZATION_MAIN_FILE.value
    elif filename == "activities.xml":
```
Do not allow external users to use different cases, only lowercase will be allowed